GVRf/Framework/jni/oculus/activity_jni.cpp

```diff
@@ -113,22 +113,11 @@ jclass GVRActivity::GetGlobalClassReference( const char * className ) const
     return gc;
 }
 
-void GVRActivity::ConfigureVrMode( ovrModeParms & modeParms )
+void GVRActivity::Configure( OVR::ovrSettings & settings )
 {
-    LOG( "GVRActivity::ConfigureVrMode");
-
-    modeParms.CpuLevel = 2;
-    modeParms.GpuLevel = 2;
-
-    // Always use 2x MSAA for now
-    app->GetVrParms().multisamples = 2;
-
-    // Always use bilinear texture filtering.
-    app->GetVrParms().textureFilter = TEXTURE_FILTER_BILINEAR;
-
-    app->GetSwapParms().WarpProgram = WP_SIMPLE;
+    LOG( "GVRActivity::Configure");
+    // leave it as the oculus defaults for now.
```

Comment:
I think you can remove modeParms.CpuLevel, modeParms.GpuLevel, textureFilter and WarpProgram settings here because by default those are the same as what you removed.

But for multisamples, I can see that it is loaded from system service and right now I can't know what it is.

To set it as what it was(2), you may need to add "settings.EyeBufferParms.multisamples = 2;" here to ensure it.